TramsDataApi/Gateways/AcademyConversionProjectGateway.cs

```diff
@@ -76,6 +76,11 @@ public async Task<PagedResult<AcademyConversionProject>> SearchProjects(int page
             {
                 var lowerDeliveryOfficers = deliveryOfficers.Select(officer => officer.ToLower());
                 academyConversionProjects = academyConversionProjects.Where(acp => lowerDeliveryOfficers.Contains(acp.AssignedUserFullName.ToLower()));
+                if (lowerDeliveryOfficers.Contains("Not assigned"))
+                {
+                    var notAssignedProjects = academyConversionProjects.Where(acp => acp.AssignedUserFullName == "" && acp.AssignedUserFullName == null);
+                    academyConversionProjects = academyConversionProjects.Concat(notAssignedProjects);
```

Comment:
Now that we are combining to sets of results you may need to order at the end rather than the beginning.

Reply:
Done